src/editorial/web/components/lines/squiggly.ts

```diff
@@ -1,2 +1,36 @@
+import { border } from "@guardian/src-foundations/palette"
+
+const wavelength = 16
+const amplitude = 3
```

Comment:
Are we able to change these values to make it look closer to the original squiggle? I'm not sure whether this will break the repetition on various devices / pixel densities...

```suggestion
const wavelength = 12
const amplitude = 2.5
```